AB_Hands_On/.ipynb_checkpoints/AB_Hands_On-checkpoint.ipynb

```diff
@@ -0,0 +1,6615 @@
+{
```

Comment:
<p>Maybe use mamba as it's just better? There's also mamba-forge.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
The video with conda is already recorded. Unless Alex plans on rerecording id say lets avoid it. Mamba is also listed as experimental, for a course like this conda will have more guaranteed longevity